guides/extend-graphql-add-field.md

```diff
@@ -0,0 +1,98 @@
+# At a glance
+
+Sometimes you only need to extend GraphQL to add a field to an existing type. Here's how to do it.
+
+## Extend the schema
+
+- If it doesn't already exist, create `schemas` folder in the plugin, and add an `index.js` file there.
```

Comment:
Since this is about extending a field we can safely assume that the files and folder exists. We can start with changes to add to the `.graphql` file

Reply:
If we are extending an already existing schema this is not true. There would be no reason to "extend" a schema that is declared in the same plugin.